spyder/widgets/helperwidgets.py

```diff
@@ -22,13 +22,16 @@
                             QSpacerItem, QStyle, QStyledItemDelegate,
                             QStyleOptionFrame, QStyleOptionViewItem,
                             QToolButton, QToolTip, QVBoxLayout,
-                            QWidget, QHBoxLayout)
+                            QWidget, QHBoxLayout, QLabel)
```

Comment:
Add `QFrame` class for the suggestion bellow to work:

```suggestion
                            QWidget, QHBoxLayout, QLabel, QFrame)
```